### PR TITLE
Fixed error due to new use of path in the export of lists

### PR DESCRIPTION
**Pull request by CPAlejandro**
Hi team, we resolve the issue #2692 with this steps:
- We change `path` to `relative_dirname` and `filename` (which is the new way to use it) and instead of concatenating name, we pass another filter passing this name to filename

### CODE REVIEW

File: public/controllers/management/components/management/ruleset/list-editor.js [hunk 1]
```diff
@@ -593,12 +593,17 @@ class WzListEditor extends Component {
...
                           await exportCsv(
-                            `/lists?path=${path}/${name}`,
+                            `/lists`,
                             [
                               {
                                 _isCDBList: true,
-                                name: 'path',
-                                value: `${path}/${name}`
+                                name: 'relative_dirname',
+                                value: `${path}`
```
Comment (Desvelao) [inline public/controllers/management/components/management/ruleset/list-editor.js:601]: If you don't need to interpolate any variable into the string, you can set the variable as `value` object key.

Comment (CPAlejandro) [reply public/controllers/management/components/management/ruleset/list-editor.js]: Well, I have applied your advice. Thank you!

**Review by Desvelao — APPROVED** [review]
LGTM!

**Review by frankeros — APPROVED** [review]
LGTM!

**Pull request merged**